src/types/sponsoredchildrenrecords.ts

```diff
@@ -0,0 +1,26 @@
+import { MyInfoField, StringValue, CodeAndDesc, MyInfoAttribute, MyInfoNotApplicable } from "./base"
+
+export type SponsoredChildFull = {
+  nric: StringValue
+  name: StringValue
+  hanyupinyinname: StringValue
+  aliasname: StringValue
+  hanyupinyinaliasname: StringValue
+  marriedname: StringValue
+  sex: CodeAndDesc
+  race: CodeAndDesc
+  secondaryrace: CodeAndDesc
+  dialect: CodeAndDesc
+  dob: StringValue
+  birthcountry: CodeAndDesc
+  lifestatus: CodeAndDesc
+  residentialstatus: CodeAndDesc
+  nationality: CodeAndDesc
+  scprgrantdate: StringValue
+}
+
+export type SponsoredChildBelow21 = SponsoredChildFull
+export type SponsoredChildAbove21 = Pick<SponsoredChildFull, 'nric'>
+
+export type MyInfoSponsoredChildrenRecords = Array<MyInfoField<SponsoredChildBelow21 | SponsoredChildAbove21 | MyInfoNotApplicable>>
```

Comment:
is this accurate - an element of this array can be `NotApplicable`? just checking, I'm ok to leave it this way to be safe if we're not sure

Reply:
I'm not 100% sure either, but assuming this is just the same MyInfo data in array form, might be best to leave it in for now.